src/Components/Footer.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { css, withStyles } from "../withStyles";
+import FlexContainer from "../Containers/FlexContainer";
+import List from "../Elements/List";
+import ListItem from "../Elements/ListItem";
+
+const Footer = ({ size = "medium", styles, ...props }) => (
+  <footer {...css(styles.footer, styles[size])} {...props}>
+    <FlexContainer direction="column" align="start">
+      <List direction="row" listType="ul">
```

Comment:
No need for listType="ul", it is ul by default

Reply:
Fixed it on my latest commit, thanks for the info @robertjarske